Wireframe/index.html

```diff
@@ -1,33 +1,59 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css" />
-  </head>
-  <body>
-    <header>
-      <h1>Wireframe</h1>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Wireframe</title>
+  <link rel="stylesheet" href="style.css" />
+</head>
+
+<body>
+  <header>
+    <h1 class="center">Wireframe</h1>
+    <p class="center">
+      Explaining some web dev terms.
+    </p>
+  </header>
+  <main>
+    <article>
+      <img src="267x150 placeholder.svg" alt="" />
```

Comment:
Can you review your <img> tags? They should always have an alt text for people using screenreaders.